.github/workflows/linux_meson.yml

```diff
@@ -102,3 +102,111 @@ jobs:
       run: |
         export OMP_NUM_THREADS=2
         python dev.py -n -j 2
+
+  prerelease_deps_coverage_64bit_blas:
+    name: Prerelease deps and Blas 64bit
+    # If using act to run CI locally the github object does not exist and the usual skipping should not be enforced
+    if: "github.repository == 'scipy/scipy' || github.repository == ''"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: recursive
+
+    - name: Setup Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install Ubuntu dependencies
+      run: |
+        # NOTE: not the same OpenBLAS version as in upstream CI (I'm being lazy here)
+        sudo apt-get update
+        sudo apt-get install -y libopenblas-dev libatlas-base-dev liblapack-dev gfortran libgmp-dev libmpfr-dev libsuitesparse-dev ccache libmpc-dev
```

Comment:
@rgommers @tupui do you know why we need these: `libatlas-base-dev liblapack-dev libgmp-dev libmpfr-dev libsuitesparse-dev libmpc-dev`? Do we install these for extra coverage?

Reply:
I have no clue about what is what. I never understood it and can't find a simple explanation 😅

Reply:
ATLAS is an alternative to OpenBLAS. We need one or the other, but not both.

The others are optional dependencies - `libgmp` and `libmpfr` help with at least the test performance of `scipy.special`, because `mpmath` is glacially slow without it.